packages/samples/headless-react/src/components/smart-snippet-questions-list/smart-snippet-questions-list.class.tsx

```diff
@@ -0,0 +1,81 @@
+import {Component, ContextType} from 'react';
+import {
+  buildSmartSnippetQuestionsList,
+  SmartSnippetQuestionsList as HeadlessSmartSnippetQuestionsList,
+  SmartSnippetQuestionsListState,
+  Unsubscribe,
+} from '@coveo/headless';
+import {AppContext} from '../../context/engine';
+
+export class SmartSnippetQuestionsList extends Component<
+  {},
+  SmartSnippetQuestionsListState
+> {
+  static contextType = AppContext;
+  context!: ContextType<typeof AppContext>;
+
+  private controller!: HeadlessSmartSnippetQuestionsList;
+  private unsubscribe: Unsubscribe = () => {};
+
+  componentDidMount() {
+    this.controller = buildSmartSnippetQuestionsList(this.context.engine!);
+    this.updateState();
+
+    this.unsubscribe = this.controller.subscribe(() => this.updateState());
+  }
+
+  componentWillUnmount() {
+    this.unsubscribe();
+  }
+
+  private updateState() {
+    this.setState(this.controller.state);
+  }
+
+  render() {
+    if (!this.state) {
+      return null;
+    }
+
+    const {questions} = this.state;
+
+    if (questions.length === 0) {
+      return <div>Sorry, no related questions found</div>;
+    }
+
+    return (
+      <div style={{textAlign: 'left'}}>
+        People also ask:
+        <dl>
```

Comment:
Interesting tag choices, first time I learn of them. Are `dl`, `dd` they recommended for this kind of UI? I'm not clear on what the difference between a description list and unordered is

Reply:
I thought it fits well for a quick sample demo.

OOTB, it indents the answer under each question.